macros/get_metafields.sql

```diff
@@ -1,6 +1,38 @@
-{% macro get_metafields(source_object, reference_value, lookup_object="stg_shopify__metafield", key_field="metafield_reference", key_value="value", reference_field="owner_resource") %}
+{% macro get_metafields(source_object, reference_values=None, lookup_object="stg_shopify__metafield", key_field="metafield_reference", key_value="value", reference_field="owner_resource") %}
```

Comment:
See my point above. I don't believe we should be including a default value for `reference_values` as there wouldn't be a time where we want that to not be present.

Reply:
Agreed. Updated.